XCUITests/ToolbarTest.swift

```diff
@@ -112,18 +112,20 @@ class ToolbarTests: BaseTestCase {
             waitForExistence(app.otherElements["Navigation Toolbar"])
         }
         navigator.openURL(website1["url"]!, waitForLoading: true)
+        // Adding the waiter right after navigating to the webpage in order to make the test more stable
+        waitUntilPageLoad()
+        waitForExistence(app.buttons["TabLocationView.pageOptionsButton"])
         let pageActionMenuButton = app.buttons["TabLocationView.pageOptionsButton"]
-        waitForExistence(pageActionMenuButton, timeout: 5)
         let statusbarElement = app.statusBars.children(matching: .other).element.children(matching: .other).element(boundBy: 0)
         XCTAssertTrue(statusbarElement.isHittable)
         app.swipeUp()
         let hiddenStatusbarElement = app.statusBars.children(matching: .other).element.children(matching: .other).element(boundBy: 0)
```

Comment:
Trying to debug would like to know which element is this...could you add a comment to make that easier to understand? Thanks!

Reply:
The hiddenStatusbarElement is the same as the statusbarElement "app.statusBars.children(matching: .other).element.children(matching: .other).element(boundBy: 0)" but when it is hidden. I just used the hiddenStatusbarElement variable to be easier to follow the test.